modules/light-clients/07-tendermint/client_state.go

```diff
@@ -123,7 +123,7 @@ func (cs ClientState) Validate() error {
 	}
 
 	if err := light.ValidateTrustLevel(cs.TrustLevel.ToTendermint()); err != nil {
-		return err
+		return errorsmod.Wrapf(ErrInvalidTrustLevel, "%s", err)
```

Comment:
added this error wrapping for debugging ux

Reply:
Just a nit, but I think in other places we do something just like...

```suggestion
		return errorsmod.Wrap(ErrInvalidTrustLevel, err.Error())
```